.yarnrc.yml

```diff
@@ -0,0 +1,7 @@
+compressionLevel: 0
+
+enableGlobalCache: true
+
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-2.3.3.cjs
```

Comment:
Yarn is checked inside the repository, that's how we recommend to use it for now (although we're working with Node to streamline this in the future). This way all your contributors are sure to use exactly the Yarn version you want them to, without having to install it first.